messages.js

```diff
@@ -49,20 +49,24 @@ exports.init = function(sbot) {
     },
 
     tombstoneFeed(metafeedKeys, previous, feedKeys, reason, cb) {
-      let query = and(author(metafeedKeys.id), type('metafeed/add'))
+      let query = and(
+        author(metafeedKeys.id),
+        type('metafeed/add')
+      )
 
       // FIXME: getJITDB() is not a public API
       sbot.db.getJITDB().all(query, 0, false, false, (err, results) => {
         if (err) return cb(err)
+        results = results.filter(x => x.value.content.subfeed === feedKeys.id)
```

Comment:
In a way it would be great if we had a simple operator in jitdb like slowEqual that we could use for this kind of filter where we have 90% of the query covered by indexes already.